client/src/components/jwt.tsx

```diff
@@ -0,0 +1,28 @@
+import UserContext from '../context/UserContext';
+import { 
+	buildRequestParams 
+} from './RequestBuilder';
+
+export const validate = async (user:any) => {
+	const params = buildRequestParams({
+		token: user.userData.token,
+	});
+
+	const request = await fetch(`api/validate?${params}`, {
```

Comment:
Je kan beter: API(`validate${params}`).Get() gebruiken een post is niet zo handig denk ik omdat je krijgt geen data maar een boolean.